sycl/source/CMakeLists.txt

```diff
@@ -63,6 +63,8 @@ function(add_sycl_rt_library LIB_NAME LIB_OBJ_NAME)
   else()
     target_compile_options(${LIB_OBJ_NAME} PUBLIC
                            -fvisibility=hidden -fvisibility-inlines-hidden)
+
+    target_compile_options(${LIB_OBJ_NAME} PUBLIC -Wno-psabi)
```

Comment:
Add a comment.
```
Silencing GCC note about changed ABI in quite old version of GCC which is not supported by the product:
"the ABI for passing parameters with 32-byte alignment has changed in GCC 4.6" 
```

BTW. For some reason I was not able to silence the note just using `pragma` in the files, so silencing it globally.